### PR TITLE
Update configuration for SSR with Next.js

### PR DESCRIPTION
**Pull request by pauldariye**
Update the docs for server-side-rendering with Next.js and add reference to an example from the Next.js repo.

### CODE REVIEW

File: sections/advanced/server-side-rendering.js [hunk 1]
```diff
@@ -56,34 +56,35 @@ const ServerSideRendering = () => md`
...
     }
   }
   \`\`\`
-
-  Here we're wrapping the \`Main\` component, which contains the Next.js app, and are using this
-  to extract the styles on the server side.
+ 
+  > Refer to [Next.js example `with-styled-components`](https://github.com/zeit/next.js/tree/master/examples/with-styled-components)
 
   > This is unfortunately only a workaround! It will accumulate rules over time, so you will need
```
Comment (philpl) [inline sections/advanced/server-side-rendering.js]: We should be able to remove this note now :smile:

File: sections/advanced/server-side-rendering.js [hunk 2]
```diff
@@ -56,34 +56,35 @@ const ServerSideRendering = () => md`
...
         </html>
       )
     }
   }
   \`\`\`
-
-  Here we're wrapping the \`Main\` component, which contains the Next.js app, and are using this
-  to extract the styles on the server side.
+ 
+  > Refer to [Next.js example `with-styled-components`](https://github.com/zeit/next.js/tree/master/examples/with-styled-components)
```
Comment (philpl) [inline sections/advanced/server-side-rendering.js]: * the

And this won't need to be a note (Remove `> `)

Also, maybe we can make this a little nicer? "Refer to [our example] in the next.js repo for an up-to-date usage example."

Comment (pauldariye) [reply sections/advanced/server-side-rendering.js]: Thanks, @philpl. I've updated the language.

File: sections/advanced/server-side-rendering.js (hunk 1 above)
Comment (pauldariye) [reply sections/advanced/server-side-rendering.js]: Removed

File: sections/advanced/server-side-rendering.js (hunk 2 above)
Comment (pauldariye) [reply sections/advanced/server-side-rendering.js]: I've removed the leading `>`. Should be good now.

**Comment by philpl** [comment]
Thanks a lot for fixing this up :tada:

**Pull request merged**